pkg/e2e/verify/namespace_webhook.go

```diff
@@ -194,17 +194,21 @@ var _ = ginkgo.Describe(namespaceWebhookTestName, func() {
 				for privilegedNamespace := range PRIVILEGED_NAMESPACES {
 					err := e2eCreatePrometheusRule(ctx, prometheusName, privilegedNamespace, privilegedUser, "", h)
 					Expect(err).NotTo(HaveOccurred())
+					err = deletePrometheusRule(privilegedNamespace, prometheusName, h)
+					Expect(err).NotTo(HaveOccurred())
 				}
 				for _, namespace := range NONPRIV_NAMESPACES {
 					err := e2eCreatePrometheusRule(ctx, prometheusName, namespace, privilegedUser, "", h)
 					Expect(err).NotTo(HaveOccurred())
+					err = deletePrometheusRule(namespace, prometheusName, h)
+					Expect(err).NotTo(HaveOccurred())
 				}
 			}
 		}, viper.GetFloat64(config.Tests.PollingTimeout))
 
 		util.GinkgoIt("Non-privileged users can create prometheusrule all non-privileged namespaces", func(ctx context.Context) {
 			for _, nonPrivilegedNamespace := range NONPRIV_NAMESPACES {
-				err := e2eCreatePrometheusRule(ctx, prometheusName, nonPrivilegedNamespace, DUMMY_USER, "dedicated-admins", h)
+				err := e2eCreatePrometheusRule(ctx, prometheusName, nonPrivilegedNamespace, DUMMY_USER, "DUMMY_GROUP", h)
```

Comment:
```suggestion
				err := e2eCreatePrometheusRule(ctx, prometheusName, nonPrivilegedNamespace, DUMMY_USER, DUMMY_GROUP, h)
```

I think you are meaning to use the `DUMMY_GROUP` variable here and not a string of "DUMMY_GROUP"

Reply:
Hey @reetika-vyas this hasn't been resolved